crates/sui-graphql-rpc/src/test_infra/cluster.rs

```diff
@@ -33,8 +33,8 @@ use tokio::task::JoinHandle;
 use tokio_util::sync::CancellationToken;
 use tracing::info;
 
-const VALIDATOR_COUNT: usize = 7;
-const EPOCH_DURATION_MS: u64 = 15000;
+const VALIDATOR_COUNT: usize = 4;
```

Comment:
Why this change?

Reply:
The default for all of our other tests is 4, so 7 just results in more load during the test for little value since we aren't even interested in testing the validator portion of the stack. I found that this helped reduced timeouts when these tests were run when the GitHub actions runner was under high load.